test/unit/test_operationfactory.cpp

```diff
@@ -5871,8 +5882,10 @@ TEST(
         EXPECT_EQ(list[0]->nameStr(),
                   "Inverse of NAD83(CSRS)v6 to CGVD28 height (1) + "
                   "NAD83(CSRS)v6 to CGVD2013(CGG2013) height (1) "
-                  "using Ballpark geographic offset "
-                  "from NAD83(CSRS) to NAD83(CSRS)v6");
+                  "using Null geographic offset "
```

Comment:
Changed the test to pass after EPSG update. But it seems to be wrong.

Reply:
I believe this is correct. This is due to the "Canada - CGVD28" extend being extend to New Foundland, which adds a new transformation for it. The previous one using the ballpark geographic offset is then sorted in second position. But the test should be modified to ASSERT_GE(list.size(), 2U); and check list[1] instead. Checking the updated list[0] doesn't hurt either.

Reply:
done